src/features/listings/utils/submissionFormSchema.ts

```
@@ -31,7 +31,21 @@ const submissionSchema = (
         ])
         .optional(),
       otherInfo: z.string().optional(),
-      ask: z.union([z.number().int().min(0), z.null()]).optional(),
+      ask: z
+        .union([
+          z
+            .number()
+            .min(0)
+            .refine(
+              (val) => {
+                const decimalPlaces = (val.toString().split('.')[1] || '').length;
```

ESLint

[error] Insert `⏎··················`

(prettier/prettier)